discovery/chan_series.go

```diff
@@ -321,20 +321,21 @@ func (c *ChanSeries) FetchChanUpdates(chain chainhash.Hash,
 		return nil, err
 	}
 
-	// msgFlags indicates the presence of optional fields in a channel update.
-	// Zero indicates no optional fields.
-	msgFlags := lnwire.ChanUpdateFlag(0)
-
 	chanUpdates := make([]*lnwire.ChannelUpdate, 0, 2)
 	if e1 != nil {
+		msgFlags := 1
```

Comment:
this needs a comment

Reply:
Removed this since we added `MessageFlags` to `ChannelEdgePolicy`, so we should just be able to use that instead.